### PR TITLE
Updating GitHub actions to cut a release

### PR DESCRIPTION
**Pull request by dmurphy18**
### What does this PR do?
Updating GitHub actions to cut a release

### What issues does this PR fix or reference?
Need to cur a release using GitHub actions for GitHub repository

### Previous Behavior
not applicable

### New Behavior
Ability to cur a release, note, moving from GitLab use of release directory and version schematics , to dated version similar to Bootstrap-salt.sh|.ps1 versioning, e.g. v2024.11.29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,21 @@ The script can install, remove, and check the status of an installed Salt minion
 either using a direct command line option or via VMware's use of Guest Variables,
 commonly referred to as guestVars.
 
+In every two-step installation example, you would be well-served to **verify against the SHA256
+sum** of the downloaded ``svtminion.sh`` file.
+
+.. _sha256sums:
+
+The SHA256 sum of the ``svtminion.sh`` file, per release, is:
```
Comment (twangboy) [inline README.md:22]: Will this get auto-populated by the release?

**Review by twangboy — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -14,6 +14,21 @@ The script can install, remove, and check the status of an installed Salt minion
 either using a direct command line option or via VMware's use of Guest Variables,
 commonly referred to as guestVars.
 
+In every two-step installation example, you would be well-served to **verify against the SHA256
+sum** of the downloaded ``svtminion.sh`` file.
+
+.. _sha256sums:
```
Comment (ScriptAutomate) [inline README.md:20]:
```suggestion
## sha256sums
```

Suggested update from `rST` to Markdown.

Not sure if Markdown usage impacts anything like the functionality of `.github/workflows/scripts/update-release-shasum.py`

File: README.md [hunk 3]
```diff
@@ -14,6 +14,21 @@ The script can install, remove, and check the status of an installed Salt minion
 either using a direct command line option or via VMware's use of Guest Variables,
 commonly referred to as guestVars.
 
+In every two-step installation example, you would be well-served to **verify against the SHA256
+sum** of the downloaded ``svtminion.sh`` file.
```
Comment (ScriptAutomate) [inline README.md:18]:
```suggestion
sum** of the downloaded `svtminion.sh` file.
```

Updating from `rST` to Markdown.

File: README.md (hunk 1 above)
Comment (ScriptAutomate) [inline README.md:22]:
```suggestion
The SHA256 sum of the `svtminion.sh` file, per release, is:
```

Updating from `rST` to Markdown.

**Review by ScriptAutomate — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dmurphy18) [reply README.md:20]: using the actions from bootstrap-salt, and it looks for the .. _sha256sums:, will change the python code in update-release-shasum.py to look for ```## sha256sums``` and update the README.md for that marker

File: README.md (hunk 1 above)
Comment (dmurphy18) [reply README.md:22]: will do, reckoned on some changes from rst to md

File: README.md (hunk 3 above)
Comment (dmurphy18) [reply README.md:18]: done

**Review by ScriptAutomate — APPROVED** [review]

**Pull request merged**